### PR TITLE
[BB-6988] Publish to pypi

### PR DESCRIPTION
**Pull request by pkulkark**
## Description

This PR updates the requirements to the correct repos and adds the publish workflow to release the package to PyPI. I have built the wheel locally and verified that it contains all the important files as mentioned in #98.

### CODE REVIEW

File: poll/settings.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
 # http://django-statici18n.readthedocs.io/en/latest/settings.html
 
 with open(os.path.join(BASE_DIR, 'poll/translations/config.yaml'), 'r') as locale_config_file:
-    locale_config = yaml.load(locale_config_file)
```
Comment (xitij2000) [inline poll/settings.py:68]: I think we can use SafeLoader here?

**Review by xitij2000 — APPROVED** [review]
:+1: Looks good! Just had a few questions.

- [x] I tested this: approved based on CI testing 
- [x] I read through the code
- [na] I checked for accessibility issues
- [na] Includes documentation
- [na] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.

File: Makefile [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 .DEFAULT_GOAL := help
 
-FIREFOX_VERSION := "94.0.1"
```
[the remark this answers is not in the document]
Comment (pkulkark) [reply Makefile:5]: Yup but that's not compatible with the selenium version used in xblock-sdk (test suite depends on it). This is the latest version that is compatible.

File: poll/settings.py (hunk 1 above)
Comment (pkulkark) [reply poll/settings.py:68]: You're right. Updated it.

**Pull request merged**